local-cli/__tests__/bundle-test.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+var Promise = require('promise');
+var Bundle = require('../../packager/react-packager/src/Bundler/Bundle');
+
+jest.mock('fs');
```

Comment:
Apparently fs is not auto mocked because it's a node module. That's why I am explicitly mocking it here.